gallery/communicationservices-insights/Microsoft.Communication-CommunicationServices.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@yves-chan we'll have to add a "communicationservices-insights" insights entry to insightsportal